org.adempiere.base/src/org/compiere/model/MTreeFavoriteNode.java

```diff
@@ -0,0 +1,121 @@
+/******************************************************************************
+  Copyright (C) 2017 Adaxa Pty Ltd. All Rights Reserved.                
+  This program is free software; you can redistribute it and/or modify it    
+  under the terms version 2 of the GNU General Public License as published   
+  by the Free Software Foundation. This program is distributed in the hope   
+  that it will be useful, but WITHOUT ANY WARRANTY; without even the implied 
+  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *****************************************************************************/
+
+package org.compiere.model;
+
+import java.sql.ResultSet;
+import java.util.Properties;
+
+import org.adempiere.exceptions.AdempiereException;
+import org.compiere.util.DB;
+import org.compiere.util.Env;
+import org.compiere.util.Msg;
+
+/**
+ * Favorite Tree Node Model
+ * 
+ * @author Logilite Technologies
+ * @since  June 20, 2017
+ */
+public class MTreeFavoriteNode extends X_AD_Tree_Favorite_Node
+{
+
+	/**
+	 * 
+	 */
+	private static final long	serialVersionUID		= -1085269880909860587L;
+
+	public static final String	SQL_CHECK_MENU_EXISTS	= "SELECT COUNT(AD_Menu_ID) > 0 FROM AD_Tree_Favorite_Node WHERE AD_Tree_Favorite_ID=? AND Parent_ID=? AND AD_Menu_ID=?";
```

Comment:
This syntax is not valid in Oracle:
ORA-00923: FROM keyword not found where expected

Just use ANSI SQL syntax.